oasislmf/utils/deterministic_loss.py

```diff
@@ -60,12 +60,12 @@ def generate_deterministic_losses(
     ri = any(re.match(r'RI_\d+$', fn) for fn in os.listdir(input_dir))
 
     step_flag = ''
-    if 'fm_profile.csv' in os.listdir(input_dir):
-        with open(os.path.join(input_dir, 'fm_profile.csv')) as f:
-            reader = csv.reader(f)
```

Comment:
It is probably best to avoid using the `csv` lib. - Pandas is the best for processing CSVs, even for simple reading and writing.